substratevm/src/com.oracle.graal.pointsto/src/com/oracle/graal/pointsto/meta/AnalysisField.java

```diff
@@ -51,6 +51,8 @@
 import jdk.vm.ci.meta.ResolvedJavaField;
 import jdk.vm.ci.meta.ResolvedJavaType;
 
+import static com.oracle.graal.pointsto.util.AtomicUtils.atomicMark;
```

Comment:
No static imports please. The downsides (not immediately clear which method is called at the call site) do not justify saving a few characters.

Reply:
Okay, done.